app/index.html

```diff
@@ -17,9 +17,9 @@
   <header class="hero">
     {% include 'templates/topnav.html' %}
     <div class="wrapper">
-      <h1 class="hero-title">Polymer Project</h1>
+      <div class="hero-title">Polymer Project</div>
```

Comment:
I updated H tags (H1, H2, H3) styles so that the landing pages & blogs can have consistent/harmonic type scale. 

h1 { font-size: 40px; }
.hero-title { font-size: 72px; }

The hero-title is much larger than h1, and not used in blog posts. The title in blog post is h1(see attached image), for simplicity, i think it's better to keep .hero-title independent of the H tag. 


<img width="912" alt="screen shot 2018-11-26 at 4 37 50 am" src="https://user-images.githubusercontent.com/406636/49014509-389a5800-f135-11e8-9127-22465d50e070.png">